digits/templates/models/images/classification/show.html

```diff
@@ -117,18 +122,21 @@ <h4 class='text-center'>Dataset</h4>
 updateSnapshotList({% autoescape false %}{{task.snapshot_list()}}{% endautoescape %});
                             </script>
                         </div>
+                    </div>
+                    <div class="col-sm-4">
                         <button
                             formaction="{{url_for('models_download', job_id=job.id())}}"
                             formmethod="post"
                             formenctype="multipart/form-data"
                             class="btn btn-info">
-                            Download
+                            Download Model
                         </button>
                     </div>
                 </div>
                 {% if task.get_framework_id() in framework_ids %}
```

Comment:
Alternatively, perhaps we need to move this line up (before the model selection button)?